tests/Fixer/LanguageConstruct/SingleSpaceAfterConstructFixerTest.php

```diff
@@ -3193,4 +3193,47 @@ public function provideFixWithSwitchCases(): \Generator
             ',
         ];
     }
+
+    /**
+     * @requires PHP 8.0
```

Comment:
Why not running this test on PHP 7.4?

Reply:
this is indeed not needed, thanks! removed :+1: